app/assets/stylesheets/application.scss

```diff
@@ -83,6 +83,10 @@ $thredded-button-font-size: 1.25rem;
   min-height: 100vh;
 }
 
+#main-footer {
+  margin: 0;
+}
+
 #error_explanation {
```

Comment:
I noticed that this id is the only one which uses underscores instead of dashes in this file. Could fix as a separate issue if we want to be really strict with naming conventions.